src/components/EditPage.js

```diff
@@ -0,0 +1,39 @@
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+import React from "react";
+import { Link } from "react-router-dom";
+
+const GET_SINGLE_PAGE = gql`
+  query getSingleServicePage($pageSlug: String) {
```

Comment:
I thought I would be able to query by slug and discussed why I couldn't get ID working here:
https://github.com/cityofaustin/techstack/issues/436#issuecomment-398224605